scripts/upgradeable/upgradeable.patch

```diff
@@ -202,12 +137,12 @@ index d94e956a..86bb5713 100644
  import {IERC5267} from "../../interfaces/IERC5267.sol";
  
  /**
-@@ -30,27 +29,19 @@ import {IERC5267} from "../../interfaces/IERC5267.sol";
+@@ -29,28 +28,18 @@ import {IERC5267} from "../../interfaces/IERC5267.sol";
+  * separator from the immutable values, which is cheaper than accessing a cached version in cold storage.
   *
   * _Available since v3.4._
-  *
+- *
 - * @custom:oz-upgrades-unsafe-allow state-variable-immutable state-variable-assignment
```

Comment:
The patch removes this from the vanilla version ... but if we don't want it in the upgradeable version, why is it in the vanilla version in the first place ?

Reply:
I think this is a mistake, we indeed want the annotation as per [this comment](https://github.com/OpenZeppelin/openzeppelin-contracts/pull/3969#issuecomment-1420145285). We may have just not ported it to upgradeable contracts

Reply:
This comment is duplicate, discussing in https://github.com/OpenZeppelin/openzeppelin-contracts/pull/4424#discussion_r1253074417